_example/basic/go.mod

```diff
@@ -0,0 +1,12 @@
+module main
+
+go 1.17
```

Comment:
lets change this to `go 1.16` .. its a strange thing where this specifies the min go version which is compat

Reply:
done